pkg/controller/pods.go

```diff
@@ -92,29 +101,39 @@ func (cont *AciController) checkIfEpgExistPod(pod *v1.Pod) {
 	}
 
 	key := cont.aciNameForKey("pod", podkey)
-	epGroup := pod.ObjectMeta.Annotations[metadata.EgAnnotation]
-	notExist, egval := cont.checkVrfCache(epGroup, "pod[EpgAnnotation]")
-
-	if notExist && egval.Name != "" {
-		desc := "Pod Annotation failed for the pod " + pod.Name + "(Namespace " +
-			pod.Namespace + ", Reason being: " + "Cannot resolve the EPG " + egval.Name +
-			" for the tenant " + egval.Tenant + " and app-profile " + egval.AppProfile
-		faultCode := strconv.Itoa(10)
-		severity := "major"
-		cont.createFaultInst(key, desc, faultCode, severity)
+	if len(pod.ObjectMeta.Annotations[metadata.EgAnnotation]) > 0 {
```

Comment:
Please change this line 
epGroup, ok  := pod.ObjectMeta.Annotations[metadata.EgAnnotation]
if ok {
  cont.handleEpgAnnotationUpdate(key, 10, major, pod.Name, epGroup)
}